democracy/views/contact_person.py

```diff
@@ -15,26 +23,12 @@ class Meta:
 
     def create(self, validated_data):
         if not validated_data["organization"]:
-            validated_data["organization"] = self.context['request'].user.get_default_organization()
+            validated_data["organization"] = self.context["request"].user.get_default_organization()
         return super().create(validated_data)
 
 
 class ContactPersonViewSet(viewsets.ReadOnlyModelViewSet, mixins.CreateModelMixin, mixins.UpdateModelMixin):
     serializer_class = ContactPersonSerializer
     queryset = ContactPerson.objects.select_related("organization").order_by("name")
-    permission_classes = (permissions.IsAuthenticatedOrReadOnly,)
+    permission_classes = (ContactPersonPermission,)
```

Comment:
Related to my other comment on `ContactPersonPermission`, you could do this instead:
```suggestion
    permission_classes = [permissions.IsAuthenticated, ContactPersonPermission]
```
Or:
```suggestion
    permission_classes = [permissions.IsAuthenticated & ContactPersonPermission]
```